drivers/storage/portworx/portworx.go

```diff
@@ -663,14 +664,29 @@ func SetPortworxDefaults(toUpdate *corev1.StorageCluster) {
 		}
 	}
 
+	// Enable CSI if running in k3s environment or
+	// if this is a new Portworx installation
 	if t.isK3s || toUpdate.Spec.Version == "" {
```

Comment:
what's the reason to only do this for new installation? IMO this should be done on all installations.

Reply:
Simplified logic and existing customers might be using the feature gate w/ Gitops. If a customer is using GitOps and we keep removing the feature flag, gitops will keep adding it back in.

Also keeps the behavior of enabling CSI by default only for new installations.

Reply:
after reading your next comment, I agree - will make this change.